lib/ntbb-session.lib.php

```diff
@@ -475,7 +475,13 @@ function getAssertion($userid, $serverhostname, $user = null, $challengekeyid =
 		if (($user['userid'] === $userid) && !empty($user['loggedin'])) {
 			// already logged in
 			$usertype = '2';
-			if (in_array($userid, $psconfig['sysops'], true)) {
+			$isupperstaff = $curuser['group'] === 6 || $curuser['group'] === 2;
+			$ismain = $serverhostname === 'sim3.psim.us' || $serverhostname === 'smogtours.psim.us';
+			if (
+				in_array($userid, $psconfig['sysops'], true) ||
+				// give admins and leaders sysop on side servers
```

Comment:
no

Reply:
I really really don't like it when people "sneak" major policy changes in like this. Please actually say if you do something like this in the pull request description.

I think I would be amenable to this if it was discussed first.

Reply:
Oh, sorry. I was told this was good to go. 
CC: Aeonic

Reply:
OK, after upper staff discussion, I'm fine with this. The main thing I dislike is that this is nowhere in the pull request description, like you were trying to hide it from me.

It might be useful to compare to what it looks like when I make a pull request:

https://github.com/smogon/pokemon-showdown/pull/8267

I don't need _exactly_ this much detail, but you should at least describe any change with privacy or permissions implications, as well as the specific features being added (this would also prevent my confusion over why this isn't in `servers/server.php`).

Reply:
Sorry; this change skipped my mind when I was making the PR description. I'll keep it more detailed in the future.